module/Applications/config/console.config.php

```diff
@@ -3,7 +3,7 @@
  * YAWIK
  *
  * @filesource
- * @copyright (c) 2013 - 2020 Cross Solution (http://cross-solution.de)
+ * @copyright (c) https://yawik.org/LICENSE.php
```

Comment:
Shouldn't that be `COPYRIGHT.php`?
Also the "(c)" is not needed anymore.
That's true for all other changes below...